Test/git-issues/git-issue-3187a.dfy.expect

```diff
@@ -1,4 +1,4 @@
-git-issue-3187a.dfy(5,12): Error: a const field should be initialized using ':=', not '='
-git-issue-3187a.dfy(7,12): Error: a local variable should be initialized using ':=', ':-', or ':|', not '='
-git-issue-3187a.dfy(8,22): Error: a variable in a let expression should be initialized using ':=', ':-', or ':|', not '='
+git-issue-3187a.dfy(5,10): Error: a const field should be initialized using ':=', not '='
```

Comment:
That error is so much more accurate, thanks !

Reply:
Thanks